redskyapi/experiments/v1alpha1/api.go

```diff
@@ -520,7 +533,7 @@ func (h *httpAPI) GetAllExperimentsByPage(ctx context.Context, u string) (Experi
 }
 
 func (h *httpAPI) GetExperimentByName(ctx context.Context, n ExperimentName) (Experiment, error) {
-	u := h.client.URL(endpointExperiment + url.PathEscape(n.Name()))
+	u := h.client.URL(endpointExperiment + n.Name())
```

Comment:
The escaping was not necessary here; at first I thought this was the bug (e.g. I was seeing `%2520` in the effective URL).